docs/source/instrument/_install_new_instrument.rst

```diff
@@ -0,0 +1,301 @@
+Installation
```

Comment:
change title to `Install New Instrument` so the sidebar entry matches

extend the underline to match new title length

Reply:
The rest of this document looks great!